.git_archival.txt

```diff
@@ -0,0 +1 @@
+ref-names: $Format:%D$
```

Comment:
@Nothing4You FYI the recent versions of setuptools-scm need a more complete template — the current one has 4 lines, not just one.